TimePlatformer/Assets/Scripts/PlayerController.cs

```diff
@@ -58,7 +77,7 @@ void Update ()
         if (Input.GetKeyDown(KeyCode.Space))
         {
             TimeController.Instance.ToggleTime();
-        }
+        } 
```

Comment:
Don't include changes like this that have no effect on the code. There should never be a whitespace ending a line.